data/abandonment-config.schema.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Exceptions for abandoned packages",
+  "description": "Community-sourced overrides for packages that appear abandoned but are still maintained",
+  "type": "object",
+  "definitions": {
+    "datasourceConfig": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      }
+    }
+  },
+  "properties": {
```

Comment:
This seems non-ideally static. I'd prefer to keep it simple with just any string allowed and rely on human review, than to have it needing to be statically maintained